lms/templates/emails/reverification_processed.txt

```diff
@@ -1,43 +1,27 @@
-    ${_("Your verification for course {course_name} and assessment {assessment} "
-        "has been passed."
-    ).format(course_name=course_name, assessment=assessment)}
+${_("We have successfully verified your identity for the {assessment} assessment in the {course_name} course. To go to the courseware, select the following {course_link}."
+).format(course_name=course_name, assessment=assessment, course_link=course_link)}
 
-%else:
-    ${_("Your verification for course {course_name} and assessment {assessment} "
-        "has failed."
-    ).format(course_name=course_name, assessment=assessment)}
+% elif is_attempt_allowed and verification_open:    # if attempts are allowed and verification is still open.
+${_("We could not verify your identity for the {assessment} assessment in the {course_name} course. You have used {used_attempts} out of {allowed_attempts} attempts to verify your identity."
+).format(course_name=course_name, assessment=assessment, used_attempts=used_attempts, allowed_attempts=allowed_attempts)}
+%if due_date:
+${_("You must verify your identity before the assessment closes on {due_date}.").format(due_date=due_date)}
+%endif
 
-    % if not is_attempt_allowed:
-        ${_("You have reached your allowed attempts limit. No more retakes allowed.")}
-    % elif not verification_open:
-        ${_("Assessment date has passed and retake not allowed.")}
-    % else:
-        % if due_date:
-            ${_("Assessment closes on {due_date}.".format(due_date=due_date))}
-        % else:
-            ${_("Assessment is open and you have {left_attempts} attempt(s) remaining.".format(left_attempts=left_attempts))}
-        % endif
+${_("To try to verify your identity again, select the following {}.").format(reverify_link)}
 
-        ${_("Click on link below to re-verify:")}
-        % if is_secure:
-          https://${ site }${ reverify_link }
-        % else:
-          http://${ site }${ reverify_link }
-        % endif
+% elif not is_attempt_allowed or not verification_open:    # if attempts are not allowed or verification window is closed
+${_("We could not verify your identity for the {assessment} assessment in the {course_name} course. You have used {used_attempts} out of {allowed_attempts} attempts to verify your identity, and verification is no longer possible."
+).format(course_name=course_name, assessment=assessment, used_attempts=used_attempts, allowed_attempts=allowed_attempts)}
 
-    % endif
-% endif
+%endif
 
-    ${_("Click on link below to go to the courseware:")}
-    % if is_secure:
-      https://${ site }${ courseware_url }
-    % else:
-      http://${ site }${ courseware_url }
-    % endif
+${_("To go to the courseware, select the following {}.").format(course_link)}
 
+${_("If you have any questions, you can contact student support {support_link}.").format(support_link=support_link)}
 
-
-${_("The {platform_name} Team.").format(platform_name=platform_name)}
+${_("Thanks,")}
+${_("The edX team")}
```

Comment:
We shouldn't hardcode "edX" here.  Please use the PLATFORM_NAME setting (retrieved via microsites)